### ISSUE TITLE
rc2 postback very slow on first execution

### ISSUE THREAD

**Issue by azimmerer** [aspnet/Mvc#4814]
I recently upgraded my asp.net 5 application to asp.net core 1.0.

The postbacks are very slow, it takes 45 seconds for the postback objected to be created and till the first breakpoint in the postback method is reached. After that the breakpoint is hit and the request is executed normaly. If I execute the same method again, the postback is very fast.
It only happens if I post back objects created by the entity framework with dependencies to other object.

For Instance:

```c#
public class PostbackObject
{ 

public virtual Object1 Object1 {get;set;}
public int Object1ID {get;set;}
}
```

If I comment out Object1

```c#
public class PostbackObject
{ 

//public virtual Object1 Object1 {get;set;}
public int Object1ID {get;set;}
}
```

I don't have any issues.

**Comment by davidfowl** [comment]
See https://github.com/aspnet/Mvc/issues/4666

**Comment by Eilon** [comment]
@rynowak does this sound the same as #4666 and this is a dup?

**Comment by azimmerer** [comment]
It is the same. The fix from #4666 worked for me. Thank you.
